src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -323,6 +325,28 @@ protected boolean isFinished() {
             }
         };
     }
+    public Command GoToTopLift() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+                disengageBrake();
+                ShiftToHighGear();
+            }
+
+            @Override
+            protected void execute() {
+                if topLimitSwitch.get == true {
+                    liftDrive.set (.5);
+                }   else return
+
+            }
+
+            @Override
+            protected boolean isFinished() {
+                engageBrake();
```

Comment:
This function should be returning true or false to indicate the command is done. engageBrake() should go in end(), as well as a call to liftDrive.set(0) or something similar to ensure we stop the motor.